draft-ietf-mls-architecture.md

```diff
@@ -815,14 +816,22 @@ MLS requires the following policies to be defined, which restrict the set of
 acceptable behavior in a group. These policies must be consistent between
 deployments for them to interoperate:
 
+- A policy on which ciphersuites are acceptable.
+- A policy on any mandatory or forbidden MLS extensions.
 - A policy on when to send proposals and commits in plaintext instead of
   encrypted.
 - A policy for which proposals are valid to have in a commit, including but not
   limited to:
   - When a member is allowed to add or remove other members of the group.
   - When, and under what circumstances, a reinitialization proposal is allowed.
-  - When proposals from external senders are allowed.
-  - When external joiners are allowed.
+  - When proposals from external senders are allowed and how to authorize
+    those proposals.
+  - When external joiners are allowed and how to authorize those external
+    commits.
+  - Which other proposal types are allowed.
+- A policy of when members should commit pending proposals in a group.
+- A policy of how to protect and share the GroupInfo objects needed for
+  external joins.
```

Comment:
This is dup with line 776

Reply:
I did not read line 776 to include the concerns I had in mind in 833-834. For example, in a federated environment perhaps non-transport level encryption of the GroupInfo from the DS to the client that wants to join is required.

Reply:
I would just consider encryption or no encryption part of the mechanism in line 776

Reply:
Honestly I read this section of the document several times and never got that impression. A lot of people are going to miss something that subtle and implicit. Even if some people might interpret line 776 as including this, what harm is there in making it clear in line 883-884?